### PR TITLE
added option (default=false) to email subject beautifier to include nvr

### LINKED ISSUE #1635: Put versions back into e-mail subjects
In https://github.com/fedora-infra/bodhi/pull/1561 we made it so update e-mail subjects use the new beautify title code (which is very nice), but inadvertently we also stopped including version info on the packages and now only list the package names. It has been requested that we bring back the versions into e-mail subjects, so we should modify the new beautify title to have an ```nvr``` flag (defaulting to ```False```) that includes the ```nvr``` of the Build if ```True```, and uses the current behavior if ```False```.

### PR DESCRIPTION
**Pull request by brandongray**
fixes #1635

### CODE REVIEW

File: bodhi/server/models.py [hunk 1]
```diff
@@ -1653,7 +1653,11 @@ def beautify_title(self, amp=False):
         &amp; html entity
         """
```
Comment (bowlofeggs) [inline bodhi/server/models.py]: Let's document the new nvr parameter in this docblock.

File: bodhi/server/models.py [hunk 2]
```diff
@@ -1653,7 +1653,11 @@ def beautify_title(self, amp=False):
...
         if len(self.builds) > 2:
```
Comment (bowlofeggs) [inline bodhi/server/models.py]: You could simplify this a bit by introducing an inline function for how to pick the label to do a little less if/elsing. Something like this:

```
def build_label():
    return build.nvr if nvr else packagename_from_nvr(self, build.nvr)
```

Then you can just change the calls to ```packagename_from_nvr``` to ```build_label```.

It's fine to keep it the way it is if you prefer, just a suggestion.

File: bodhi/tests/server/test_models.py [hunk 3]
```diff
@@ -1078,15 +1078,21 @@ def test_beautify_title(self):
...
 
         update.builds.append(rpm_build)
         eq_(update.beautify_title(), u'TurboGears, TurboGears, and 1 more')
+        eq_(update.beautify_title(nvr=True),
+            u'TurboGears-1.0.8-3.fc11, TurboGears-1.0.8-3.fc11, and 1 more')
 
         p = HTMLParser()
         eq_(p.unescape(update.beautify_title(amp=True)), u'TurboGears, TurboGears, & 1 more')
+        eq_(p.unescape(update.beautify_title(amp=True, nvr=True)),
+            u'TurboGears-1.0.8-3.fc11, TurboGears-1.0.8-3.fc11, & 1 more')
```
Comment (bowlofeggs) [inline bodhi/tests/server/test_models.py:1095]: Nice!

**Review by bowlofeggs — CHANGES_REQUESTED** [review]
The only required change to merge is to document the new parameter in the docblock. Thanks for the patch!

File: bodhi/server/models.py (hunk 2 above)
Comment (brandongray) [reply bodhi/server/models.py]: I like this idea!

**Comment by brandongray** [comment]
Ah, rebased wrong! I'm going to close this PR and try again.

**Pull request closed without merging**